alibi/explainers/cfrl_base.py

```diff
@@ -177,13 +180,13 @@ def sample(self) -> Dict[str, Optional[np.ndarray]]:
         rand_idx = np.random.randint(low=0, high=self.len * self.batch_size, size=(self.batch_size,))
 
         # Extract data form buffers.
-        X = self.X[rand_idx]                                        # input array
-        Y_m = self.Y_m[rand_idx]                                    # model's prediction
-        Y_t = self.Y_t[rand_idx]                                    # counterfactual target
-        Z = self.Z[rand_idx]                                        # input embedding
-        Z_cf_tilde = self.Z_cf_tilde[rand_idx]                      # noised counterfactual embedding
-        C = self.C[rand_idx] if (self.C is not None) else None      # conditional array if exists
-        R_tilde = self.R_tilde[rand_idx]                            # noised counterfactual reward
+        X = self.X[rand_idx]  # input array
+        Y_m = self.Y_m[rand_idx]  # model's prediction
+        Y_t = self.Y_t[rand_idx]  # counterfactual target
+        Z = self.Z[rand_idx]  # input embedding
+        Z_cf_tilde = self.Z_cf_tilde[rand_idx]  # noised counterfactual embedding
+        C = self.C[rand_idx] if (self.C is not None) else None  # conditional array if exists
+        R_tilde = self.R_tilde[rand_idx]  # noised counterfactual reward
```

Comment:
I can re-align the comments if desired, my IDE collapses everything when I format...

Reply:
I'm also one for aligning nicely, but all IDE's seem to disagree with us...